client/components/plan-storage/index.jsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+import React from 'react';
+import { connect } from 'react-redux';
+import noop from 'lodash/noop';
+
+/**
+ * Internal dependencies
+ */
+import QueryMediaStorage from 'components/data/query-media-storage';
+import { getMediaStorage } from 'state/sites/media-storage/selectors';
+import PlanStorageButton from 'components/plan-storage/button';
+
+const PlanStorage = React.createClass( {
+
+	displayName: 'PlanStorage',
```

Comment:
The `displayName` attribute is unnecessary as it will be inferred from the const name.